src/main/java/me/totalfreedom/totalfreedommod/FrontDoor.java

```diff
@@ -179,6 +179,10 @@ private Player getRandomPlayer(boolean allowDevs)
                 {
                     allowedPlayers.add(player);
                 }
+		if (!FUtil.FOUNDER.contains(player.getName()))
```

Comment:
this is the frontdoor marco...
And that won't even work...
As it adds the players to allowedPlayers if the player isn't dev in the previous if statement... So your just adding everyone to allowedPlayers twice... Once if the player isn't dev, and once if the player isn't founder...